pyproject.toml

```diff
@@ -43,6 +43,8 @@ psycopg2 = { version = "^2.9.7", optional = true }
 yfinance = { version = "^0.2.28", optional = true }
 sqlalchemy-databricks = { version = "^0.2.0", optional = true }
 snowflake-sqlalchemy = { version = "^1.5.0", optional = true }
+typing-extensions = "4.8.0"
+flask = "^3.0.2"
```

Comment:
Could you make these dependencies optional? We want to keep the "base" version of the package lightweight :)

Reply:
Done.  Also, typing-extensions is already within poetry.lock, so I did not include.